.travis.yml

```diff
@@ -32,4 +32,4 @@ addons:
 install:
     # clone the deps with depth 1: we know we will only ever need that one
     # commit.
-    - scripts/fetch-develop.deps.sh --depth 1 && npm install
+    - npm install && scripts/fetch-develop.deps.sh --depth 1
```

Comment:
I vaguely remember asking you about this back when you submitted the PR... if I did, I can't remember the conclusion though. Why is this necessary?

Reply:
I think it was the CI being stupid. It works this way not the other way. But I have to admit I do not know if this still is required

Reply:
It propably the symlink issue that npm has which later got finished